Apache/Solr/Document.php

```diff
@@ -298,8 +298,7 @@ public function getFieldValues()
 	 * }
 	 * </code>
 	 */
-	#[\ReturnTypeWillChange]
-	public function getIterator()
+	public function getIterator(): Traversable
```

Comment:
Need to add `use Traversable;`

Reply:
This barbaric class isn't namespaced, so I don't need to import Traversable.

Reply:
Oh. That's sad lol but okay